models/public/face-recognition-resnet34-arcface/model.yml

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  Deep face recognition net with ResNet34 backbone and Arcface loss <https://arxiv.org/abs/1801.07698>
+task_type: face_recognition
+files:
+  - name: model-r34-arcface-ms1m-refine-v1.zip
```

Comment:
Please add sizes for all files (they're required since a recent downloader change).